### PR TITLE
Improve lab components visibility on the site

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
- Update copy on components overview page
- Add "In progress" status to lab components page heading
- Add 🚧 to sidebar label for lab components

### CODE REVIEW

**Review by lilyvc — DISMISSED** [review]

**Review by joshwooding — COMMENTED** [review]
I thought the table on the page was also being removed?

**Review by joshwooding — COMMENTED** [review]
@mikearildbrown The copy for labs is pretty hidden on the page and is easily missed

**Comment by JoanaMMoreira** [comment]
> I thought the table on the page was also being removed?

I thought we were keeping it until we have the docs for ag grid theme but I can remove it if that's not the case

**Comment by JoanaMMoreira** [comment]
> it looks like the sort order is the same, is this something we can change?

@joshwooding did we agree we would make the change in Mosaic rather than changing the order for all core component pages individually?

**Review by libertymayc — APPROVED** [review]

File: site/docs/components/combo-box/index.mdx [hunk 1]
```diff
@@ -26,6 +28,7 @@ data:
       { name: "List", relationship: "contains" },
       { name: "Input", relationship: "contains" },
     ]
+  status: In progress
```
Comment (origami-z) [inline site/docs/components/combo-box/index.mdx:31]: Probably something more clear to lab, e.g., "Lab component"

Comment (origami-z) [reply site/docs/components/combo-box/index.mdx:31]: in progress doesn't tell direct association to lab component, especially "do not use in production". It's likely different than the "In process" in foundation actual mean

Comment (JoanaMMoreira) [reply site/docs/components/combo-box/index.mdx:31]: @mikearildbrown @origami-z is there a consensus on this? I personally like "in progress" and I think the sidebar icon provides enough context but let me know either way so we can get this merged :)

**Review by feliciackh — APPROVED** [review]

**Pull request merged**